manifests/0000_50_cluster-storage-operator_01_operator_namespace.yaml

```diff
@@ -1,3 +1,4 @@
+# Create the namespace at runlevel 49, so cluster-csi-snapshot-controller-operator (running at level 50) can use it.
```

Comment:
To create the NS at runlevel 49, shouldn't you rename it to `manifests/0000_49_cluster-storage-operator_01_operator_namespace.yaml` instead?

Reply:
Fixed